src/test/regress/bin/diff

```diff
@@ -12,7 +12,10 @@
 
 file1="${@:(-2):1}"
 file2="${@:(-1):1}"
-test=$(basename $file1 .out)
+# $file1 is the expected file, which can have _0 suffix, so we use results file
```

Comment:
We also have an expected file with `custom_aggregate_support_1.out` and could potentially have more, so maybe use any integer regex instead of `_0`?

Reply:
And, also the sentence is a bit hard to follow. We're discussion only `file1 ` in the text, but sed uses `file2`. Is it a typo?

Reply:
Changed the regex to match any sequence of digits, and updated the comment.